### PR TITLE
ariane -> cva6

### PR DESCRIPTION
**Pull request by faddat**
**Related issue**:  #606 

**Type of change**:  other enhancement

**Impact**:  other

**Release Notes**
Renamed Ariane to CVA6

### CODE REVIEW

File: .circleci/defaults.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ mapping["chipyard-boom"]=" CONFIG=SmallBoomConfig"
 mapping["chipyard-blkdev"]=" CONFIG=SimBlockDeviceRocketConfig"
 mapping["chipyard-hwacha"]=" CONFIG=HwachaRocketConfig"
 mapping["chipyard-gemmini"]=" CONFIG=GemminiRocketConfig"
-mapping["chipyard-ariane"]=" CONFIG=ArianeConfig"
+mapping["chipyard-cva6"]=" CONFIG=Cva6Config"
```
Comment (colinschmidt) [inline .circleci/defaults.sh:59]: I think every place we have `Ariane` we should have `CVA6` as that seems to be their preferred capitalization.

File: generators/firechip/src/main/scala/TargetConfigs.scala [hunk 2]
```diff
@@ -179,10 +179,10 @@ class SupernodeFireSimRocketConfig extends Config(
   new FireSimRocketConfig)
 
 //**********************************************************************************
-//* Ariane Configurations
+//* CVA6 Configurations
 //*********************************************************************************/
-class FireSimArianeConfig extends Config(
+class FireSimCva6Config extends Config(
```
Comment (jerryz123) [inline generators/firechip/src/main/scala/TargetConfigs.scala:184]: This requires changing the default build recipes defined in FireSim: 
https://github.com/firesim/firesim/blob/master/deploy/sample-backup-configs/sample_config_build_recipes.ini

File: .circleci/defaults.sh (hunk 1 above)
Comment (faddat) [reply .circleci/defaults.sh:59]: Sure, I can do that.

File: generators/firechip/src/main/scala/TargetConfigs.scala (hunk 2 above)
Comment (faddat) [reply generators/firechip/src/main/scala/TargetConfigs.scala:184]: OK.  

I think the whole thing requires changing the wrapper for Ariane, as well, right?

Comment (jerryz123) [reply generators/firechip/src/main/scala/TargetConfigs.scala:184]: Yes, that submodule would need to be updated and bumped as well. Our standard routine is to open PRs for and submodules which need to be bumped, and link to them in this PR. Then we merge everything together once everything is approved.

**Comment by alonamid** [comment]
subsumed by #710

**Pull request closed without merging**